tests/account_ecosystem_derived/ecosystem_user_id_lookup_v1/test_script/telemetry_stable.account_ecosystem_v4.yaml

```
@@ -0,0 +1,3 @@
+- submission_timestamp: 2020-04-01 22:09:03.243074 UTC
+  ecosystem_user_id: a0
+  previous_ecosystem_user_ids: null
```

Does this need to be something like?
```suggestion
  payload:
    ecosystem_user_id: a0
    previous_ecosystem_user_ids: null
```